spec/lib/site_settings/validations_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+require 'site_settings/validations'
+
+describe SiteSettings::Validations do
+  subject { Class.new.include(described_class).new }
+
+  context "s3 buckets reusage" do
+    let(:error_message) { I18n.t("errors.site_settings.s3_bucket_reused") }
+
+    shared_examples "s3 bucket validation" do
+      def change_other_bucket_value(value)
```

Comment:
hmm I would just call this `change_bucket_value`. The word `other` isn't really reflective of any actions being down in this method.